addons/redhat/QUALITY.md

```diff
@@ -0,0 +1,30 @@
+# Regression testing
+Regressions will be run periodically by QE, in order to ensure that everything is functional.  This should be done before every release, including new builds of previous releases.  Nothing should be delivered to customers that QE hasn't run a regression on and signed off on.
```

Comment:
Who is upstream QE here ?    our upstream CI  should be configured to  run regression  ., I don't think QE  will be running any regression on upstream